internal/admin/dashboard/templates/page-usage.html

```
@@ -0,0 +1,207 @@
+            </div>
+        </template>
+    </div>
+
+    <!-- Request Log Section -->
+    <div class="usage-log-section">
+        <h3>Request Log</h3>
+        <div class="usage-log-toolbar">
+            <div class="usage-filter-row usage-filter-row-search">
+                <div class="filter-input-wrap">
+                    <i data-lucide="search" class="filter-input-icon" aria-hidden="true"></i>
+                    <input type="text" placeholder="Search by request ID, model, provider..." x-model="usageLogSearch" @input.debounce.300ms="fetchUsageLog(true)" class="filter-input">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)